pypesto/variational/pymc.py

```diff
@@ -0,0 +1,190 @@
+import logging
```

Comment:
Module-level docstrings would be nice. Short descriptions of what all the `.py` files are there for. Good example in `pypesto.store` modules